esi_leap/db/api.py

```diff
@@ -114,6 +114,11 @@ def offer_verify_availability(offer_ref, start, end):
         offer_ref, start, end)
 
 
+def offer_verify_node_availability(resource_node_uuid, start, end):
```

Comment:
I'd use `resource_availability` and `resource_uuid`

Reply:
resource_node_uuid makes more sense because we are passing in a uuid and not an availability.